gradle/dependencies.gradle

```diff
@@ -24,6 +24,10 @@ def versions = [
     support: '26.1.0'
 ]
 
+def apt = [
+        autoService : "com.google.auto.service:auto-service:1.0-rc3",
```

Comment:
nit: make this match the rest of this file's style

Reply:
also rc4 is out

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/05a3a34b3778da0031e5dfa0161256f224ed6cdf